### DISCUSSION TITLE
咋每次发了个新版都就有问题了

### DISCUSSION THREAD

**Discussion by jetBn** [web-infra-dev/rsbuild#1514] [Q&A]
![image](https://github.com/web-infra-dev/rsbuild/assets/26957220/3f9afc60-3f0f-4d63-b50e-702cd7029d4f)

**Comment by jetBn** [comment]
使用jenkins 应该是在 linux. 系统上

**Comment by chenjiahan** [comment]
麻烦提供下复现方式

**Reply by jetBn** [reply]
就是本地电脑 我的mac M1 没事 正常打包  但是 放在jenkins上就不行了 。感觉跟系统有关系

**Reply by jetBn** [reply]
然后服务器上打包也有问题，感觉是不知道哪个依赖装的有问题 跟本地装的不一样可能。
